### DISCUSSION TITLE
Does this library have signals and slots mechanic like Qt?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/eepp#63] [General]
Or a messaging system like the [FOX Toolkit](https://en.wikipedia.org/wiki/Fox_toolkit)?

**Comment by SpartanJ** [comment]
No signals or slot mechanics (I don't like it). Just messaging system with event listeners and callbacks. You can check the [examples](https://github.com/SpartanJ/eepp/tree/develop/src/examples/7guis) to understand how it works.
